BREAKING_CHANGES.md

```diff
@@ -0,0 +1,12 @@
+# Breaking Changes
+
+## 1.0.0
```

Comment:
Since the API contract isn't officially released yet, I suggest you stay at version 0.X for now. Just bump the minor version.